src/Maestro/Client/src/Generated/BuildTime.cs

```diff
@@ -0,0 +1,118 @@
+using System;
```

Comment:
Missing header.

Reply:
This is a generated file. My understanding is that we do not have the header on any of the generated files.

Reply:
Whoops.  Wasn't paying attention.